Protos/fetch.sh

```diff
@@ -23,7 +23,7 @@ upstream="$here/upstream"
 checkouts="$(mktemp -d)"
 
 # Clone the grpc and google protos into the staging area.
-git clone --depth 1 https://github.com/grpc/grpc-proto "$checkouts/grpc-proto"
+git clone --depth 2 https://github.com/grpc/grpc-proto "$checkouts/grpc-proto"
```

Comment:
Why did the depth change to 2?

Reply:
Thanks for pointing this out, I left it by mistake in the PR. (it's from something I tried but wasn't necessary)